web_client/views/ControlWidget.js

```diff
@@ -42,7 +42,7 @@ var ControlWidget = View.extend({
         if (options && options.norender) {
             return this;
         }
-        this.$el.html(this.template()(this.model.attributes));
+        this.$el.html(this.template()(this.model.attributes));  // eslint-disable-line backbone/no-view-model-attributes
```

Comment:
`this.model.toJSON()` is the correct way to get a simple object of a Model's attributes.